ibm/mas_devops/roles/ocp_provision/README.md

```diff
@@ -31,6 +31,12 @@ Flag that determines if GPU worker nodes should be added during cluster creation
 - Environment Variable: `OCP_PROVISION_GPU`
 - Default Value: `false`
 
+### gpu_wpool_name
+The name of the gpu worker pool to added to or modify in the cluster. If already existing, use the existing name to avoid recreating another gpu worker pool unless that is the goal. 
+
+- Environment Variable: `GPU_WPOOL_NAME`
+- Default Value: `gpu`
+
 ### gpu_workers
 The number of GPU worker nodes to deploy in the cluster. Depends on `ocp_provision_gpu` and is currently only set up for ROKS clusters.
```

Comment:
Could we add the flavour (mg4c.32x384.2xp100), to this description, that these worker nodes would be so users don't have to look in the code to see (or see after provision)

Reply:
done